producers.md

```diff
@@ -242,9 +244,40 @@ subcomponents of either components or production components.
 A subcomponent inherits all bindings from its parent component, and so it is
 often a simpler way of building nested scopes.
 
-### Logging and debugging
+### Monitoring
 
-==As of April 2015, not implemented yet==
+[ProducerMonitor][ProducerMonitor] can be used to monitor the execution of
+producer methods; its methods correspond to various places in a producer's
+lifecycle.
+
+To install a `ProducerMonitor`, contribute into a set binding of
+[ProductionComponentMonitor.Factory][ProductionComponentMonitorFactory]. For
```

Comment:
Have to resolve separately, as this is a sync CL.  It's a good note, though.  Feel free to throw it into a Pull Request, if you wish, as we can yank pull requests in more easily now.  But these sync pulls I don't want to alter directly, in that they reflect a 1:1 with already recorded commits internally.